crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -1305,12 +1305,22 @@ impl Account {
                 // we might try to create the same session again.
                 // TODO: separate the session cache from the storage so we only add
                 // it to the cache but don't store it.
-                store
-                    .save_changes(Changes {
-                        sessions: vec![result.session.clone()],
-                        ..Default::default()
-                    })
-                    .await?;
+                let mut changes =
+                    Changes { sessions: vec![result.session.clone()], ..Default::default() };
+                // Any new Olm session will bump the Olm wedging index for the
```

Comment:
This seems to be also wrapped manually, though it doesn't look as comical as the other doc comment, wouldn't hurt to let `cargo fmt` format this as well.